Sources/HipMobileUI/ViewModels/NavigationViewModel.cs

```diff
@@ -44,10 +47,11 @@ public virtual void OnHidden()
         }
```

Comment:
this method does nothing?

Reply:
In this class, yes. But some of the classes inheriting from it override this method.